SlevomatCodingStandard/Sniffs/Functions/DisallowEmptyFunctionSniff.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Functions;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\FunctionHelper;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+use const T_FUNCTION;
+use const T_WHITESPACE;
+
+class DisallowEmptyFunctionSniff implements Sniff
+{
+
+	public const CODE_EMPTY_FUNCTION = 'FunctionMissingComment';
```

Comment:
Please change the value too: "EmptyComment"

Reply:
You mean probably EmptyFunction, not empty comment

Reply:
You're right :)